src/movableCoord.js

```diff
@@ -667,22 +667,22 @@ MovableCoord에 대한 상세한 내용은 데모를 살펴보기 바란다.
 		},
 
 		// set up 'css' expression
-		_reviseOptions: function(options) {
+		_reviseOptions: function() {
 			var key;
+			var self = this;
 			$.each(["bounce", "margin", "circular"], function(i, v) {
```

Comment:
You used `forEach` in `infiniteGrid`. [See it](https://github.com/naver/egjs/pull/330/files#diff-eaa190073eec9fe40432c1fa47c064f3R292)
Is there any reason can not use 'forEach`?